contracts/HypercertMinterV0.sol

```diff
@@ -91,10 +122,18 @@ contract HypercertMinterV0 is
         require(account != address(0), "Mint: mint to the zero address");
 
         // Parse data to get Claim
-        (Claim memory claim, string memory _uri, bytes32 claimHash) = _bytesToClaimAndURI(data);
+        (Claim memory claim, string memory _uri) = _bytesToClaimAndURI(data);
+
+        for (uint256 i = 0; i < claim.impactScopes.length; i++) {
```

Comment:
Note: Check whether passing storage structs to in-contract functions is the most gas efficient pattern. Alternatively, OpenZeppelin also offers Enumerable structs but their data-types might not fit our needs.

P.S. certainly not a showstopper now imho

Reply:
A quick search turns up comments talking about potential issues from passing around structs with contract size limits, but not much about gas costs. I guess we'd need to do some testing ourselves.

Reply:
On option is to not use a seperate method and just access the struct directly per for loop

Reply:
ok, inlining the fn does appear to save gas

pre:
<img width="811" alt="hypercerts_pre_inline" src="https://user-images.githubusercontent.com/640958/186658235-0484d42e-dc46-43f2-b968-c60cfc9446b4.png">

post:
<img width="809" alt="hypercerts_post_inline" src="https://user-images.githubusercontent.com/640958/186658230-54614e50-6283-47b6-80f9-46ab9d76f45e.png">

Reply:
Appreciate the thorough investigation. Indeed it's a little more optimised and I see you refactored methods to inline 👍